Cloud changelog March 24th (#4390)
mattermost/docs: source/cloud/cloud-administration/cloud-changelog.md

This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.

## Release 2021-03-24

### Improvements

#### User Interface (UI)

>  - Added a UI field for restricted domains under **Site Configuration > Posts** in the System Console. Expanded the logic that determines whether a post has a preview or not.

Reviewer: @amyblais Can you tell me which feature PR this is driven by, please? (It's not yet on my radar.)

Author: @cwarnermm https://github.com/mattermost/mattermost-server/pull/16869

>  - Added support for auto-RTL detection in browsers.

Reviewer: ```suggestion
 - Added support for automatic right-to-left (RTL) detection in browsers.
```

 - Updated the font size for the **Add People** channel modal.

>  - Online status is now shown in the channel switcher.

Reviewer: @amyblais May I have the feature PR driving this line too, please? Thanks!

Author: @cwarnermm https://github.com/mattermost/mattermost-webapp/pull/6609

 - Improved the design and layout of email notifications.

#### Administration
 - Profiling the Mattermost server with pprof is now available for Team Edition.
 - Added attributes to split.io feature flags.

### Bug Fixes
 - Fixed bugs related to replication lag for Enterprise Edition instances configured to use read replicas.

>  - Fixed an issue where Compliance report field headers were not correctly aligned.

Reviewer: ```suggestion
 - Fixed an issue where Compliance Report field headers were not correctly aligned.
```

 - Fixed an issue where the ``/join`` command was case-sensitive.
 - Fixed an issue where one-character sidebar category names were not displayed.

>  - Fixed an issue with a theme discrepancy on close buttons on some modals in the System Console (when using custom team theme).

Reviewer: ```suggestion
 - Fixed an issue with a theme discrepancy on close buttons on some modals in the System Console (when using a custom team theme).
```

>  - Fixed an issue where the text draft area in the right-hand side was jumpy with long text input when selected.

Reviewer: ```suggestion
 - Fixed an issue where long text input in the right-hand pane was jumpy when selected.
```

This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.

## Release 2021-03-24

### Improvements

#### User Interface (UI)
 - Added support for automatic right-to-left (RTL) detection in browsers.
 - Updated the font size for the **Add People** channel modal.
 - Online status is now shown in the channel switcher.

>  - Improved the design and layout of email notifications.

Reviewer 2: let's be specific about which types of email notifications were updated, not all types were. It should be in the PR

Author: Updated.

>  - Improved the design and layout of email notifications for password resets, member invites, and member welcome and verifications.

Reviewer: ```suggestion
 - Improved the design and layout of email notifications for password resets, member invites, member welcome, and verifications.
```

#### Administration

>  - Added mmctl commands to create, list, download and delete export files.

Reviewer: ```suggestion
 - Added ``mmctl`` commands to create, list, download, and delete export files.
```
